### PR TITLE
Adds the site nav to the home page

### PR DESCRIPTION
**Pull request by GBKS**
Which makes the sub-pages accessible from the home page.

Big step, as it makes the guide accessible for all site visitors. Let's keep this as a draft and get the site in a shape where everybody is happy before we merge this.

### CODE REVIEW

**Comment by pavlenex** [comment]
concept ACK. It's easier to test stuff locally when having proper navigation so it's useful and can always be ironed out later on imo.

File: contribute.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Contribute
 description: Find out how to get involved in Bitcoin design
 permalink: /contribute/
 main_nav: true
-nav_order: 5
+nav_order: 2
 main_classes: -no-top-padding
```
Comment (ConorOkus) [inline contribute.md:8]: Feels a bit weird to have a reference to a CSS class in here? Can it be moved to the Guide layout at all?

**Comment by danielnordh** [comment]
ACK!
This makes the site feel more real, we still won't promote the guide (as it's not yet at MVP) but it lets people follow the progress easily.

**Comment by ConorOkus** [comment]
ACK

**Pull request merged**